Testnet/NFTStore/NFTStore/NFTStore.cs

```diff
@@ -173,4 +174,10 @@ public struct SaleInfo
         public ulong Price;
         public Address Seller;
     }
+
+    public struct ReceiveInfo
+    {
+        public Address Contract;
```

Comment:
Could there be use for an index here at all?

Reply:
This is not a log. It is just struct saved in PersistentState